src/platforms/java/common/performance/instrumentation/open-feign.mdx

```diff
@@ -14,7 +14,7 @@ Capturing transactions requires that you first <PlatformLink to="/performance/">
 
 </Note>
 
-Sentry OpenFeign integration provides the `SentryFeignClient`, which creates a span for each outgoing HTTP request executed with a [Feign](https://github.com/OpenFeign/feign)-based HTTP client.
+Sentry OpenFeign integration provides the `SentryFeignClient`, which creates a scope for each outgoing HTTP request executed with a [Feign](https://github.com/OpenFeign/feign)-based HTTP client.
```

Comment:
`span` should be correct here. If you look at the code here (https://github.com/getsentry/sentry-java/blob/81a1a6c19ce01552f9a7f0c8814398e573877c86/sentry-openfeign/src/main/java/io/sentry/openfeign/SentryFeignClient.java#L52) you can see how it works.